Dockerfile

```diff
@@ -1,15 +1,19 @@
 ARG BUILD_TYPE=dev
-ARG GOLANG=golang:1.19
-ARG BASE=fedora:36 
+# TODO(katexochen)
+ARG BUILDER_BASE=ghcr.io/katexochen/golang-fedora:36
```

Comment:
Should we push it to quay.io/kata-containers? maybe even having a CI that pushes it?